pages/index.html

```diff
@@ -1,12 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8"/>
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet" />
+  <title>Melley Gebretatios</title>
 </head>
 <body>
-  
-</body>
+  <section class="container">
+  <header class="cell-1">
+    <nav>
+      <ul id="main-menu">
+        <li>
+          <a class="button" href="index.html">Home</a>
+        </li>
+        <li>
+          <a  class="button" href="about.html">About</a>
+        </li>
+        <li>
+          <a class="button" href="portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+    <section class="cell-2" id="middle--image">
+      <img src="../assets/IMG_9896.jpg" alt="Girl wearing a jeans top">
+    </section>
+    <section class="cell-3" id="middle--intro">
+      <h2> Hello! I'm Melley.Welcome!</h2>    
+    </section>
+  <footer class="cell-5" > 
+    <section id="footer--copyright">&copy; Copyright 2022</section>
+    <section id="footer--contact"> &copy; contactme@gmail.com</section>
```

Comment:
Instead, let's make these list items in an unordered list and style the `li` element inside the `footer` rather than giving them each an `id`